lib/util.mjs

```diff
@@ -47,8 +48,31 @@ function printWindows(windowTitle, windows, totalLength = 30, windowLength = DEF
   }
 }
 
+// Parse a string like "3/11/20" to a Temporal.Date
+// This is complicated: see https://github.com/tc39/proposal-temporal/issues/515
+function parseMDYToTemporal(mdyString) {
+  // Step 1: parse the date using the JS Date constructor.
+  // jsDate is set to midnight on the parsed date in my current time zone.
+  const jsDate = new Date(mdyString);
+
+  // Step 2: convert to a Temporal.Absolute.
+  const absolute = Temporal.Absolute.fromEpochMilliseconds(jsDate);
+
+  // Step 3: convert to a Temporal.DateTime using my current time zone.
+  const dateTime = absolute.inTimeZone(Temporal.now.timeZone());
```

Comment:
So curious. The data we are working with and showing is very specific to NYC, and I think show the data as of NYC makes a lot of sense. is there a best practice here when it comes to INTL?

Reply:
The problem is that the CSV file has date formats like "3/11/20", which is recognized by the JavaScript Date constructor but not by Temporal.  However, JavaScript Date needs a time component, and when parsing a date without a time, it sets the time to midnight in the system time zone ([spec](https://tc39.es/ecma262/#sec-date-year-month-date-hours-minutes-seconds-ms)):

> Set O.[[DateValue]] to TimeClip(UTC(finalDate)).

where UTC "converts t from local time to UTC".

When converting to Temporal, I need to undo what the Date constructor did, which is why I passed in the system time zone here.

However, in general, better practice might be to write a custom parser that extracts the year, month, and day out of the y/m/d format, and then pass the results into Temporal.Date.from().  Then you don't depend on the fragile behavior of the JavaScript Date constructor.